test/runners/typeql_runner.py

```diff
@@ -93,39 +92,28 @@ def run_failing_queries(self, queries: List[str], type: TransactionType) -> str:
                 return FailureMode.Commit
         return FailureMode.NoFailure
 
-    def run_transaction(self, queries: List[str], type: TransactionType, counted=False, rollback=False, documents=False) -> Union[int, None]:
-        count_var_name = "automatic_test_count"
-        if counted:
-            queries[-1] = queries[-1] + f"\nreduce ${count_var_name} = count;"
+    def run_transaction(self, queries: List[str], type: TransactionType, rollback=False) -> Union[int, None]:
```

Comment:
Note: removed parallel/optimized execution (shouldn't be needed in docs...)

Simplified with new ability to count fetch queries as well